### DISCUSSION TITLE
Skipping skullstripping

### DISCUSSION THREAD

**Discussion by Mak2055** [nipreps/nibabies#306] [Q&A]
Hello all,

I have some skullstripped data that I would like to analyze using nibabies. When I flag "--skull-strip-t1w skip", I get the response that the skull stripping is currently required. Is there a way of overcoming this issue?

Thank you in advance,
M

**Comment by mgxd** [comment]
Hi M,

It is not currently implemented, but keep an eye out for future releases!

Cheers,
Mathias

**Comment by mgxd** [comment]
@Mak2055 `--skull-strip-anat` is now available in the 24.0.x series.
